dashboard/test/controllers/api/v1/schools_controller_test.rb

```diff
@@ -0,0 +1,83 @@
+require 'test_helper'
+
+class Api::V1::SchoolsControllerTest < ActionController::TestCase
+  self.use_transactional_test_case = true
+
+  ALBERT_EINSTEIN_ACADEMY_ELEMENTARY = {
+    id: 60000113717,
+    name: 'Albert Einstein Academy Elementary',
+    city: 'Santa Clarita',
+    state: 'CA',
+    zip: '91355',
+    school_district_id: 600001
+  }.deep_stringify_keys.freeze
+
+  GLADYS_JUNG_ELEMENTARY = {
+    id: 20000100207,
+    name: 'Gladys Jung Elementary',
+    city:'Bethel',
+    state: 'AK',
+    zip: '99559',
+    school_district_id: 200001
+  }.deep_stringify_keys.freeze
+
+  test 'search by school name prefix' do
+    get :search, params: {q: 'glad', limit: 40}
+    assert_response :success
+    schools = JSON.parse(@response.body)
+    assert_equal 1, schools.length
+    check_school GLADYS_JUNG_ELEMENTARY, schools[0]
```

Comment:
i think this could be simplified to e.g., `assert_equal [GLADYS_JUNG_ELEMENTARY], JSON.parse(@response.body)`

Reply:
Updated.